geniza/corpus/templates/corpus/snippets/document_option_label.html

```diff
@@ -0,0 +1,25 @@
+{# template snippet for displaying document label on a form #}
+{# used on document merge form to provide enough information to merge accurately #}
+<div class="merge-document-label">
+    <strong><a target="_blank" href="{% url 'admin:corpus_document_change' document.id %}">{{ document }}</a></strong>
+    <p>PGPID {{ document.id }}</p>
+    {% if document.description %}
```

Comment:
I think we should always have a description — and if not, it would be useful for them to know it's empty.

Reply:
Ok! I made this true for all fields, e.g. tags, notes, etc. I assumed the same would be true for those fields as well